src/main/java/ch/njol/skript/events/EvtItem.java

```diff
@@ -138,6 +133,25 @@ public class EvtItem extends SkriptEvent {
 					.since("2.8.0")
 					.requiredPlugins("Paper 1.16+");
 		}
+
+		Skript.registerEvent("Brewing Complete", EvtItem.class, BrewEvent.class, "brew[ed] [complete[d]|finish[ed]] [of %-itemtypes%]")
+			.description("Called when a brewing stand finishes brewing the ingredient and changes the potions.")
+			.examples(
+				"on brew:",
+					"\tbroadcast event-items",
+				"on brew complete of speed potion:"
+			)
+			.since("INSERT VERSION");
+
+		Skript.registerEvent("Brewing Fuel", EvtItem.class, BrewingStandFuelEvent.class, "brewing fuel[ed] [of %-itemtypes%]")
```

Comment:
```suggestion
		Skript.registerEvent("Brewing Fuel", EvtItem.class, BrewingStandFuelEvent.class, "brewing fuel[ed] [of %-itemtypes%]")
```
I'm not really a fan of this syntax at all. Maybe something more like `brew[ing [stand]] consum(e|ing) fuel` or `brewing stand fuel consumption` or something along those lines. Even that's still pretty awkward.